[2024-07-16T10:28:26.175+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-16T10:28:26.193+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_id_3.log_database_connection manual__2024-07-16T10:28:24.601752+00:00 [queued]>
[2024-07-16T10:28:26.198+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_id_3.log_database_connection manual__2024-07-16T10:28:24.601752+00:00 [queued]>
[2024-07-16T10:28:26.200+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-16T10:28:26.208+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_database_connection> on 2024-07-16 10:28:24.601752+00:00
[2024-07-16T10:28:26.212+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=461) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-16T10:28:26.213+0000] {standard_task_runner.py:63} INFO - Started process 463 to run task
[2024-07-16T10:28:26.214+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'dag_id_3', 'log_database_connection', 'manual__2024-07-16T10:28:24.601752+00:00', '--job-id', '1539', '--raw', '--subdir', 'DAGS_FOLDER/dag_id_3.py', '--cfg-path', '/tmp/tmpkh27tg5h']
[2024-07-16T10:28:26.216+0000] {standard_task_runner.py:91} INFO - Job 1539: Subtask log_database_connection
[2024-07-16T10:28:26.258+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_id_3.log_database_connection manual__2024-07-16T10:28:24.601752+00:00 [running]> on host a1689f3248db
[2024-07-16T10:28:26.335+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_id_3' AIRFLOW_CTX_TASK_ID='log_database_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-07-16T10:28:24.601752+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-16T10:28:24.601752+00:00'
[2024-07-16T10:28:26.337+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-16T10:28:26.337+0000] {resources.py:20} INFO - dag_id_3 start processing tables in database:airflow
[2024-07-16T10:28:26.337+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-16T10:28:26.338+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-16T10:28:26.347+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_id_3, task_id=log_database_connection, run_id=manual__2024-07-16T10:28:24.601752+00:00, execution_date=20240716T102824, start_date=20240716T102826, end_date=20240716T102826
[2024-07-16T10:28:26.393+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-16T10:28:26.415+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-16T10:28:26.416+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
